app/src/components/radioGroup/Radio.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {
+  Radio as AriaRadio,
+  type RadioProps as AriaRadioProps,
+} from "react-aria-components";
+import { css } from "@emotion/react";
+
+import { classNames } from "@phoenix/components";
+import { StylableProps } from "@phoenix/components/types";
+
+const baseRadioCSS = css(`
+    padding: var(--ac-global-dimension-size-50) var(--ac-global-dimension-size-100);
```

Comment:
I think we probably need radio as it's own component. Can we switch this to ToggleButton to match? https://react-spectrum.adobe.com/react-aria/ToggleButtonGroup.html